Jenkinsfile

```diff
@@ -24,6 +24,7 @@ pipeline {
         always {
           junit 'output/**/*.xml'
           cobertura autoUpdateHealth: true, autoUpdateStability: true, coberturaReportFile: 'coverage.xml', conditionalCoverageTargets: '100, 0, 0', failUnhealthy: true, failUnstable: false, lineCoverageTargets: '99, 0, 0', maxNumberOfBuilds: 0, methodCoverageTargets: '100, 0, 0', onlyStable: false, sourceEncoding: 'ASCII', zoomCoverageChart: false
+          ccCoverage("coverage.py")
```

Comment:
We probably need a config to ensure that we don't reduce our line coverage from codeclimate perspective too

Reply:
I set 75% diff coverage in the UI, as it's not possible (AFAIK) to configure coverage threshold in repo. I've also opened a ticket with codeclimate about that.

Reply:
👍